### ISSUE TITLE
stabilityai/stablelm-2-zephyr-1_6b cannot fetch tokenizer.json

### ISSUE THREAD

**Issue by mtasic85** [huggingface/candle#1653]
After building release version **stable-lm** example, I get this issue:

```
./stable-lm --cpu --use-flash-attn --temperature 0.8 --top-p 40 -n 512 --which v2-zephyr --prompt "Build me website in 10 steps:"
avx: true, neon: false, simd128: false, f16c: true
temp: 0.80 repeat-penalty: 1.10 repeat-last-n: 64
Error: request error: https://huggingface.co/stabilityai/stablelm-2-zephyr-1_6b/resolve/main/tokenizer.json: status code 404

Caused by:
    https://huggingface.co/stabilityai/stablelm-2-zephyr-1_6b/resolve/main/tokenizer.json: status code 404
```

There is not **tokenizer.json** file uploaded at https://huggingface.co/stabilityai/stablelm-2-zephyr-1_6b/tree/main

**Comment by LaurentMazare** [comment]
This will be fixed once #1654 lands but note that tokenization won't be perfect.

**Comment by LaurentMazare** [comment]
Should be all good now.

**Comment by mtasic85** [comment]
Yes, it works great.

```
./stable-lm --cpu --use-flash-attn --temperature 0.8 --top-p 40 -n 512 --which v2-zephyr --quantized --prompt "Build me website in 10 steps:"
avx: true, neon: false, simd128: false, f16c: true
temp: 0.80 repeat-penalty: 1.10 repeat-last-n: 64
tokenizer-gpt4.json [00:00:01] [███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████] 4.04 MiB/4.04 MiB 3.85 MiB/s (0s)..tablelm-2-zephyr-1_6b-q4k.gguf [00:00:15] [█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████] 883.35 MiB/883.35 MiB 57.25 MiB/s (0s)retrieved the files in 17.2447928s
loaded the model in 562.469694ms
Build me website in 10 steps: 
1. I need a simple and user-friendly design that conveys clear information about what services are being offered, what the company stands for, and how potential customers can engage with the business or organization.
2. The site should be easy to navigate, using clear menus and categories so users can easily find what they're looking for quickly.
3. This is a new venture and I need a website that showcases our brand identity in an effective manner while providing a great user experience provides all necessary information about services offered.
4. I will provide content and images for the site, but may require some design input as well to ensure it aligns with company branding and messaging.
5. I prefer a modern and clean design style, that is not overly complicated or cluttered.

Web Design-Spoken-Steps:

1. Research: Understanding the client's needs, target audience, brand identity, personality, values, and unique selling propositions in order to accurately design a website that reflects their business vision.
2. Planning: Establishing the site structure on which content will be optimized for user experience while maintaining usability; identifying essential features such giovani digital branding tone guide and brand guidelines. 
3. Conceptualization: Developing a design style but allowing room for client feedback or additional customization, developing sketches or models or wireframes to finalize the layout, interface and functionality.
4. Design: Applying the final design chosen by the client using the preferred CMS. Utilizing the best CSS framework, and advanced tools like Photoshop or Sketch.
5. Development: Using developer-friendly CMS platforms such as Wix, Squarespace or WordPress for easier development and customization. 
6. Quality Assurance and Testing: Ensuring a high-performance user experience on different devices and browsers while incorporating security features for enhanced functionality online.
7. Launch: Deploying the final website to its live state after thorough testing.

Web Design-Beneath-The-Surface:

Underneath the surface design elements, I would recommend using modern web fonts such as Roboto or Open Sans in use across the site while avoiding relying on too many fonts and their variation for a more responsive user experience. Utilizing high-quality images is key to creating an immersive digital experience.

In summary, this step-by-step plan will ensure that the website not only looks professional but also functions smoothly and provides optimal usability for clients' businesses.
472 tokens generated (12.98 token/s)
```

**Comment by mtasic85** [comment]
> This will be fixed once #1654 lands but note that tokenization won't be perfect.

I see that you used GPT4 tokenizer. Is there a way to load tokenizer using huggingface transformers library and export complete tokenizer.json?

**Comment by vincev** [comment]
If anyone is interested I implemented the [arkade100k](https://github.com/vincev/coze/blob/main/src/generator/arcade100k.rs) tokenizer following the model [python implementation](https://huggingface.co/stabilityai/stablelm-2-zephyr-1_6b/blob/main/tokenization_arcade100k.py).
